packages/vscode/src/language.mts

```diff
@@ -259,7 +261,26 @@ export class GmlProvider
   static async activate(ctx: vscode.ExtensionContext) {
     this.ctx ||= ctx;
     if (!this.provider) {
-      this.provider = new GmlProvider(await parseSpec());
+      let gmlSpecFilePath;
+      const gmlSpecSource = vscode.workspace.getConfiguration('stitch').get<string | null>('gmlSpec.source');
+      const gmlSpecFilePathFromSettings = vscode.workspace.getConfiguration('stitch').get<string | null>('gmlSpec.path');
+      if (gmlSpecSource === "external" && gmlSpecFilePathFromSettings !== null) {
+        gmlSpecFilePath = gmlSpecFilePathFromSettings;
+      } else if (gmlSpecSource === "localRuntime") {
```

Comment:
This doesn't account for the different GameMaker channels (LTS, Beta, Dev), but is a good start. One of the other Stitch packages *does* account for this, so I'll see if I can figure out a way to update this accordingly. I'll go ahead and merge it.

Reply:
This is easy to implement, but I don't know whether to obtain it automatically or manually🤨